#!/usr/bin/python3


def raise_exception_msg(message=""):
    """Gives a NameError exception and gives a message with details."""
    raise NameError(message)
